test/e2e/storage/vsphere/bootstrap.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
I don't think you should change the year of a already existing file, it should show the year the file was first added. Is that correct, @nikhita?

Reply:
Reverted the copyright year back in all the existing files.